src/app/pages/send/send.component.tsx

```diff
@@ -229,4 +246,19 @@ export class SendPage extends Component<IProps, IState> {
             this.showErrorDialog(e.toString());
         }
     }
+
+    private updateBalance() {
+        this.props.account
+            .getBalance()
+            .then(balance => {
+                this.setState({
+                    balance: this.props.account.utils.balanceToStd(balance)
+                });
+            })
+            .catch(() => {
+                this.setState({
+                    balance: 'error'
```

Comment:
as an error handling here we should hide the button if something went wrong on getBalance

Reply:
Done